Detectors/TOF/README.md

```diff
@@ -0,0 +1,2 @@
+Start on 13th April 2017
```

Comment:
IMHO this single commit does not justify a PR, I would postpone it until there is content.